test/unity/toolkits/object_detection/test_object_detector.cxx

```diff
@@ -445,6 +447,63 @@ BOOST_AUTO_TEST_CASE(test_object_detector_train) {
 
   // Deconstructing `model` here will assert that every expected call to a
   // mocked-out method has been called.
+
+  // Test Non Maximum Suppression model
```

Comment:
Let's add a new test to the test suite, instead of tacking this onto the end of `test_object_detector_train`. The code you're testing has nothing to do with training or the `train` method.

Reply:
That's a good idea, thanks.